additional-material/translations/undoing-a-commit.it.md

```diff
@@ -52,6 +47,6 @@ $ git commit -m "Continued dev"
 # Decidi di rimuovere tutto
 $ git reset --hard HEAD~2
 ```
-Il comando ```git reset --hard HEAD~2``` sposta il ramo corrente indietro di 2 commit in una sola volta annullando tutte le modifiche che hai fatto e rimuovendo i 2 *snapshots* che abbiamo appena creato nella cronologia del progetto.  
+Il comando ```git reset --hard HEAD~2``` sposta il ramo corrente indietro di 2 commit in una sola volta, annullando tutte le modifiche che hai fatto e rimuovendo i 2 *snapshots* che abbiamo appena creato nella cronologia del progetto.  
 
-P.s. Non eseguire mai ```git reset --hard``` se hai già inviato i tuoi commit ad un repository condiviso poiché causerà problemi a tutti sul quel repository.  
+P.s. Non eseguire mai ```git reset --hard``` se hai già inviato i tuoi commit ad un repository condiviso, poiché causerà problemi a tutti coloro su quel repository.  
```

Comment:
I'm not sure, but maybe 'poiché su quel repository tutti avranno dei problemi'
What do you think about?

Reply:
Indeed, that one gave me some trouble. How about:
```suggestion
P.s. Non eseguire mai ```git reset --hard``` se hai già inviato i tuoi commit ad un repository condiviso, altrimenti causerà problemi a tutti coloro che lavorano correntemente con quel repository.  
```
It's slightly longer, but clearer.